core/src/main/java/org/fakereplace/core/DefaultEnvironment.java

```diff
@@ -58,9 +58,11 @@
 
     @Override
     public boolean isClassReplaceable(String className, ClassLoader loader) {
-        if(className == null) {
-            return false; //lambdas
+        // if no packages are given, replace all classes.
+        if (replaceablePackages.length == 0) {
+            return true;
```

Comment:
I think this will cause problems. e.g. at the moment replacement for lambdas does not work properly so lambdas will break.